transforms/language/html2parquet/dpk_html2parquet/ray/transform.py

```diff
@@ -10,24 +10,14 @@
 # limitations under the License.
 ################################################################################
 
-import time
-from argparse import ArgumentParser, Namespace
-from typing import Any
 
-import pyarrow as pa
-from data_processing.runtime.pure_python.runtime_configuration import (
-    PythonTransformRuntimeConfiguration,
-)
-from data_processing.transform import AbstractTableTransform, TransformConfiguration
-from data_processing.utils import CLIArgumentProvider, get_logger
+from data_processing.utils import get_logger
 from data_processing_ray.runtime.ray import RayTransformLauncher
 from data_processing_ray.runtime.ray.runtime_configuration import (
     RayTransformRuntimeConfiguration,
 )
-from html2parquet_transform import (
-    Html2ParquetTransform,
-    Html2ParquetTransformConfiguration,
-)
+from dpk_html2parquet.transform import Html2ParquetTransform, Html2ParquetTransformConfiguration
```

Comment:
You need to run pre-commit.

Reply:
@daw3rd can you elaborate ? what is pre-commit ?